easy_my_coop_es/views/subscription_request_view.xml

```diff
@@ -1,16 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record model="ir.ui.view" id="hola">
+    <record model="ir.ui.view" id="subscription_request_form">
         <field name="name">subscription.request.form</field>
         <field name="model">subscription.request</field>
         <field name="inherit_id" ref="easy_my_coop.subscription_request_form"/>
         <field name="arch" type="xml">
             <field name="lastname" position="after">
                 <field name="vat"/>
             </field>
-            <field name="subscription_amount" position="after">
-                <field name="voluntary_contribution"/>
-            </field>
+            <field name="company_register_number" position="replace"/>
```

Comment:
It's better practice to make unused fields invisible rather than deleting them. Otherwise, subsequent modules overriding emc or emc_es and fetching  `company_register_number` w/ xpath might crash.